src/sagas/index.js

```diff
@@ -1,3 +1,32 @@
+import * as C from '../constants';
+import * as ReduxSaga from 'redux-saga';
+import * as Effects from 'redux-saga/effects';
+import * as API from '../utils/api';
+import * as AlbumActionCreators from '../action-creators/album';
+import fetchAccessToken from '../utils/fetch-access-token';
+
 export default function* root() {
+  yield [
+    ReduxSaga.takeEvery(C.ACTIONS.INITIALIZE, initialize),
+    ReduxSaga.takeEvery(C.ALBUM_ACTIONS.SAVE_ALBUM_TO_DB, saveAlbumToDB),
+    ReduxSaga.takeEvery(C.ALBUM_ACTIONS.FETCH_ALBUMS_FROM_DB, fetchAlbumsFromDB),
+  ];
+}
+
+export function* initialize() {
+  const isLoggedIn = !! fetchAccessToken();
```

Comment:
Just to coerce the access token into a boolean. This is equivalent to `Boolean(fetchAccessToken())`. But really, this should be a separate function by itself: 

```javascript
function isLoggedIn() {
  return !! fetchAccessToken();
}
```

I'll update it eventually...